students/paul_cooper/lesson_2/l2_FizBuz.py

```diff
@@ -0,0 +1,16 @@
+# this prints Fizz if the number is divisible by 3
+# this prints Buzz if the number is divisible by 5
+# this prints FizzBuzz if the number is divisible by 3 and 5
+
+for i in range(1,101):
+
+	if i%3 == 0 and i%5 == 0:
+		print('FizzBuzz')
+	elif i%5 == 0:
+		print('Buzz')
+	elif i%3 == 0:
+		print('Fizz')
+	else:
+		print(i)
+
```

Comment:
Works well, but missing some PEP8 formatting here:
Put whitespace around the modulo operator if i % 3 == 0 and ... etc.
Looks like you're using tabs, but you want to use spaces, 4 to be exact, for indentations.